### PR TITLE
Alerts strings

### PR DESCRIPTION
**Pull request by ezequielc**
Strings for Displays app Alerts.

@alex-deaconu Please review. Thanks

### CODE REVIEW

File: src/locales_json/en/displays-app.json [hunk 1]
```diff
@@ -5,6 +5,82 @@
...
+    "textFields": {
+      "description": "Description",
+      "headline": "Headline",
+      "instruction": "Instruction"
+    },
+    "textToShow": "Alert Text To Show",
+    "title": "Alerts",
+    "turnOff": "Turn Off Alerts",
+    "turnOn": "Turn On Alerts",
+    "turnOnShort": " Turn On",
```
Comment (alex-deaconu) [inline src/locales_json/en/displays-app.json]: @ezequielc is the space before `Turn On` intentional? Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc looks good. One comment. Thanks.

File: src/locales_json/en/displays-app.json (hunk 1 above)
Comment (ezequielc) [inline src/locales_json/en/displays-app.json]: @alex-deaconu No, it wasn't. Good catch! Thanks

**Comment by alex-deaconu** [comment]
@ezequielc :+1: . Thanks for fixing!

**Pull request merged**